src/cache/oss.rs

```diff
@@ -0,0 +1,41 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use opendal::layers::LoggingLayer;
+use opendal::services::Oss;
+use opendal::Operator;
+
+use crate::errors::*;
+
+pub struct OSSCache;
+
+impl OSSCache {
+    pub fn build(
+        bucket: &str,
+        key_prefix: &str,
+        endpoint: Option<&str>,
+    ) -> Result<Operator> {
+        let mut builder = Oss::default();
+        builder.bucket(bucket);
+        builder.root(key_prefix);
+
```

Comment:
It's better for us to support setting `access_key_id` here.

Reply:
Regarding the S3 configuration, I have included the `no_credentials` option. When this option is set to false, it necessitates loading two corresponding environment variables. If set to true, it should be skipped.

I believe there is no need to provide the `access_key_id` in the build function.